integration-cli/docker_api_build_test.go

```diff
@@ -16,19 +16,9 @@ import (
 
 func (s *DockerSuite) TestBuildAPIDockerFileRemote(c *check.C) {
 	testRequires(c, NotUserNamespace)
-	var testD string
-	if testEnv.DaemonPlatform() == "windows" {
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -name ba*
-RUN find /tmp/`
-	} else {
-		// -xdev is required because sysfs can cause EPERM
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -xdev -name ba*
-RUN find /tmp/`
-	}
+	testD := `FROM busybox
```

Comment:
This is the only UI change in this PR. It seems that the test checks an invalid side-effect caused by the previous implementation. I could make a fake context for this case but don't see a practical reason for it, especially after all the work that was needed in https://github.com/docker/docker/pull/31236 to hide the `Dockerfile` if no such file actually exists.

Could also add a second case that pulls a tar. Then it will have the same behavior as the previous test.